tests/e2e/specs/plugin-activation.test.js

```diff
@@ -5,27 +5,27 @@ import { deactivatePlugin, activatePlugin } from '@wordpress/e2e-test-utils';
 
 describe( 'Plugin Activation Notice', () => {
 	beforeEach( async () => {
-		await deactivatePlugin( 'google-site-kit' );
+		await deactivatePlugin( 'site-kit-by-google' );
```

Comment:
@aaemnnosttv Why did you change the slug here in this file?

Reply:
Summarizing our Slack conversation, because WordPress only uses the proper slug if it finds that plugin in an update request, let's check whether a plugin with slug `google-site-kit` is available in the list, and otherwise fall back to `site-kit-by-google`.

As soon as the plugin is launched, the latter should no longer occur, so we can remove that at some point.